package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "pipedrive",
-  "version": "1.0.1-rc.0",
-  "description": "JS API client generated by OpenAPI Generator",
-  "license": "Unlicense",
+  "version": "11.0.0-rc.1",
+  "description": "Pipedrive REST client for NodeJS",
+  "license": "MIT",
```

Comment:
I guess other fields from current package.json should be migrated to this version as well? (like repo, url, keywords etc etc)